topics/index.md

```diff
@@ -72,6 +76,13 @@ Performance
 * [valkey-benchmark](benchmark.md): The benchmarking tool shipped with Valkey.
 * [On-CPU profiling and tracing](performance-on-cpu.md): How to find on-CPU resource bottlenecks.
 
+  
+Troubleshooting
```

Comment:
Should we bundle this section with "Performance", which is talking about perf specific troubleshooting.